src/Controller/TranslateController.php

```diff
@@ -123,6 +123,14 @@ public function translate(
         $translations = $svgFile->getInFileTranslations();
         $formFields = [];
         foreach ($translations as $tspanId => $translation) {
+            // Do not display translations that are only white-space. https://stackoverflow.com/a/4167053/99667
+            // @TODO SvgFile should probably be handling this for us.
```

Comment:
If that's true, it seems like it wouldn't be too hard to put it in `hasActualTextContent()`.

Reply:
I experimented with that, but it seems that that method is only called when there's children of a text node, and isn't used to validate other node contents.

I started writing some tests so that this could be added in the right place, but went down a bit of a rabbit hole and figured the quick fix might suffice for now. I'm very happy to do the proper job though! :)

Reply:
I see. Maybe it's fine to leave it where it is for now then.